pex/common.py

```diff
@@ -15,9 +15,23 @@
 import threading
 import zipfile
 from collections import defaultdict
+from datetime import datetime
 from uuid import uuid4
 
 
+# We hardcode the datetime to January 1, 1980, which is the start of MS-DOS time. Per section 
+# 4.4.6 of the zip spec at https://pkware.cachefly.net/webdocs/casestudies/APPNOTE.TXT, 
+# Zipfiles use MS-DOS time.
+# NB: we do not respect the standard env var $SOURCE_DATE_EPOCH often used by tools for
+# deterministic timestamps, as the intended use case is security by checking if the binary was
+# tampered with; we do not have a compelling use case for respecing this env var, and it only
```

Comment:
s/respecing/respecting/

Re: the reasoning: it's more to do with the existence of the variable triggering a _reduction_ in reproducibility when pex is used on platforms that don't set the variable, or if multiple distributions were to set the env var to different values.